### ISSUE TITLE
Error installing Julia 1.7.0-beta3 on an M1 Mac

### ISSUE THREAD

**Issue by fingolfin** [johnnychen94/jill.py#83]
On an Apple Mac Mini with M1 CPU, I tried to use Jill to install Julia 1.7.0-beta3. This is with Jill 0.9.5, installed via `pip3`, using Python 3.9 as provided by homebrew. This is what happened:
```
$ python3 Library/Python/3.9/bin/jill download 1.7.0-beta3
downloading Julia release for 1.7.0-beta3-mac-arm64
failed to find 1.7.0-beta3-mac-arm64 in available upstreams. Please try it later.
```

The actual archive on the website is <https://julialang-s3.julialang.org/bin/mac/aarch64/1.7/julia-1.7.0-beta3-macaarch64.dmg>

I wonder if the mismatch `arm64` versus `aarch64` (resp. `mac-arm64` versus `macaarch64`) might be related?

**Comment by johnnychen94** [comment]
Thanks for trying this!

The error you see is duplicated to #82 and is largely a network issue.

The M1 part is not supported by jill yet because I don't have access to it. Currently, it will unconditionally point you to the x86_64 version.

Would you mind tell me what the following python code output is on your M1 machine?

```python
import platform
platform.machine()
platform.system()
platform.libc_ver()
```

**Comment by fingolfin** [comment]
```
$ python3
Python 3.9.6 (default, Jun 28 2021, 19:24:41)
[Clang 12.0.5 (clang-1205.0.22.9)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> platform.machine()
'arm64'
>>> platform.system()
'Darwin'
>>> platform.libc_ver()
('', '')
```

**Comment by fingolfin** [comment]
I am not so sure this is related to #82. The download fails no matter how often I try:
```
munk:~$ python3 Library/Python/3.9/bin/jill install 1.7.0-beta3
JILL - Julia Installer 4 Linux (MacOS, Windows and FreeBSD) -- Light

jill will:
  1) install Julia 1.7.0-beta3 for mac-arm64 into /Applications
  2) make symlinks in /Users/horn/.local/bin
You may need to manually add /Users/horn/.local/bin to PATH
Continue installation? [Y/n] y
----- Download Julia -----
downloading Julia release for 1.7.0-beta3-mac-arm64
failed to find 1.7.0-beta3-mac-arm64 in available upstreams. Please try it later.
False
munk:~$ python3 Library/Python/3.9/bin/jill install 1.7.0-beta3
JILL - Julia Installer 4 Linux (MacOS, Windows and FreeBSD) -- Light

jill will:
  1) install Julia 1.7.0-beta3 for mac-arm64 into /Applications
  2) make symlinks in /Users/horn/.local/bin
You may need to manually add /Users/horn/.local/bin to PATH
Continue installation? [Y/n] y
----- Download Julia -----
downloading Julia release for 1.7.0-beta3-mac-arm64
failed to find 1.7.0-beta3-mac-arm64 in available upstreams. Please try it later.
False
```

**Comment by johnnychen94** [comment]
Oh yes you're right! It's a separate M1 issue!

**Comment by johnnychen94** [comment]
Should be fixed in v0.9.6
